### PR TITLE
Apply linting/formatting to BoxView

### PR DESCRIPTION
**Pull request by vahidbazzaz**
This PR applies formatting and linting to BoxView as well as adding changes to the eslint and prettier configuration

Follow up: https://github.com/boxwise/boxtribute/pull/402

### CODE REVIEW

**Comment by vahidbazzaz** [comment]
[2.0 Apply linting/formatting to relevant code in FE](https://trello.com/c/qnk6KSZa)

File: react/.eslintrc [hunk 1]
```diff
@@ -5,15 +5,29 @@
...
+        "import/resolver": {
+            "node": {
+                "extensions": [".ts", ".tsx"],
+                "moduleDirectory": ["src", "node_modules"]
+            }
+        }
+    },
     "rules": {
-        "max-len": ["warn", { "ignoreComments": true }],
+        "max-len": ["warn", { "ignoreComments": true, "code": 200, "tabWidth": 4 }],
```
Comment (pylipp) [inline react/.eslintrc:21]: Hej @vahidbazzaz , can you explain why you made the adjustments to the rules? Did it turn out it does not match our requirements?

Comment (vahidbazzaz) [reply react/.eslintrc:21]: @pylipp Thank you for reviewing the PR. To make it more readable, I think we should set `code` length to `100` and reduce the `tabWidth` to 2. What do you think? I will update the documents if you agree with this change

Comment (pylipp) [reply react/.eslintrc:21]: I won't work in the FE codebase much, so I don't have that much of an opinion. Makes sense what you're saying though, go ahead!

**Review by pylipp — APPROVED** [review]

**Pull request merged**